CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Class `Scene` consisting of a background medium and structures for easier drafting and visualization of simulation setups as well as transferring such information between different simulations.
 - Solver for thermal simulation (see `HeatSimulation` and related classes).
 - Specification of material thermal properties in medium classes through an optional field `.heat_spec`.
+- Support for geometry transformations via the `Transformed` class and the helper functions `Geometry.translated`, `Geometry.scaled`, and `Geometry.rotated`.
```

Comment:
might want to also add a separate  line mentioning the `intersections_tilted_plane` or other public-facing methods introduced by this.